src/main/java/org/reaktivity/nukleus/mqtt/internal/MqttConfiguration.java

```diff
@@ -20,11 +20,15 @@
 
 public class MqttConfiguration extends Configuration
 {
+    public static final String PUBLISH_TIMEOUT_NAME = "nukleus.mqtt.publish.timeout";
+
     private static final ConfigurationDef MQTT_CONFIG;
+    public static final IntPropertyDef PUBLISH_TIMEOUT;
 
     static
     {
         final ConfigurationDef config = new ConfigurationDef("nukleus.mqtt");
+        PUBLISH_TIMEOUT = config.property("publish.timeout", 30);
```

Comment:
Suggest changing `30` to`TimeUnit.SECONDS.toSeconds(30)` to explicitly document that unit is seconds.